packages/gatsby-remark-copy-linked-files/src/index.js

```diff
@@ -32,19 +32,23 @@ const newPath = (linkNode, destinationDir) => {
   return path.posix.join(process.cwd(), DEPLOY_DIR, newFileName(linkNode))
 }
 
-const newLinkURL = (linkNode, destinationDir) => {
-  if (destinationDir) {
-    return path.posix.join(`/`, destinationDir, newFileName(linkNode))
-  }
-  return path.posix.join(`/`, newFileName(linkNode))
+const newLinkURL = (linkNode, destinationDir, pathPrefix) => {
+  const linkPaths = [`/`, pathPrefix, destinationDir, newFileName(linkNode)]
+    .filter(function(lpath) {
+      if (lpath) return true
```

Comment:
What's this for?

Reply:
If you build with `yarn develop`; the urls are correct.

When building for production, the current code does not append the `pathprefix`.

This just allows you to optionally specify a `pathPrefix`; it filters it out if `pathPrefix` happens to be empty or null.

Reply:
This produces the same result as joining paths like [gatsby-link](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-link/src/index.js#L7-L10).

However, gatsby link has to create a string, run a regex and return a new string.

The logic I have just creates the string ignoring pathPrefix if it isn't set.

Reply:
ok, so you're filtering out the sometimes empty pathPrefix 👍